README.md

```diff
@@ -1,27 +1,17 @@
-# crc [![Build Status](https://travis-ci.org/mrhooray/crc-rs.svg?branch=master)](https://travis-ci.org/mrhooray/crc-rs)
-> Rust implementation of CRC(16, 32, 64) with support of various standards
+# crc
+
+[![Build Status](https://travis-ci.org/mrhooray/crc-rs.svg?branch=master)](https://travis-ci.org/mrhooray/crc-rs)
+[![Crate](https://img.shields.io/crates/v/crc.svg)](https://crates.io/crates/crc)
+[![Docs](https://docs.rs/crc/badge.svg)](https://docs.rs/crc)
+[![License](https://img.shields.io/crates/l/crc.svg?maxAge=2592000)](https://github.com/mrhooray/crc-rs#license)
 
-* [Crate](https://crates.io/crates/crc)
-* [Documentation](https://docs.rs/crc/)
-* [Usage](#usage)
-* [Benchmark](#benchmark)
-* [License](#license)
+> Rust implementation of CRC(16, 32, 64) with support of various standards
 
 ## Usage
 Add `crc` to `Cargo.toml`
 ```toml
 [dependencies]
-crc = "^1.0.0"
-```
-or
-```toml
-[dependencies.crc]
-git = "https://github.com/mrhooray/crc-rs"
-```
-
-Add this to crate root
-```rust
-extern crate crc;
+crc = "1.8"
```

Comment:
may need to refer to latest version

Reply:
I'd wait for the newer version of the crate to be published before updating documentation. If you intend to publish the crate as soon as this is merged to master, then this should refer to "1.9"